src/sparseml/pytorch/yolov8/trainers.py

```diff
@@ -293,7 +293,8 @@ def callback_on_train_epoch_start(self):
         if model_is_quantized:
             if self.scaler is not None:
                 self.scaler._enabled = False
-            self.ema.enabled = False
+            if self.ema is not None:
+                self.ema.enabled = False
```

Comment:
pretty funny

Reply:
Yeah apparently ema is only initialized in the main process, and is none in the others!